docs/guides/ros2-gem.md

```diff
@@ -12,86 +11,151 @@ You can browse Doxygen-generated documentation on [Gem's GitHub page](https://ro
-It handles spatio-temporal relationship between this part and other frames of reference.
-It also encapsulates namespaces, which help to distinguish between different robots and different parts of the robot,
-such as in case of multiple identical sensors on one robot.
+`ROS2FrameComponent` is a representation of an interesting physical part of the robot. It handles spatio-temporal
+relationship between this part and other frames of reference. It also encapsulates namespaces, which help to distinguish
+between different robots and different parts of the robot, such as in case of multiple identical sensors on one robot.
 
 All Sensors and the Robot Control Component require `ROS2FrameComponent`.
 
 #### Sensors
 
-Sensors are Components deriving from `ROS2SensorComponent`. They acquire data from the simulated environment and publish it to ROS 2 domain.
+Sensors are Components deriving from `ROS2SensorComponent`. They acquire data from the simulated environment and publish
+it to ROS 2 domain.
+
 - Each sensor has a configuration, including one or more Publishers.
 - Sensors publish at a given rate (frequency).
 - Some sensors can be visualised.
 
-If you intend to add your own sensor, it might be useful to look at how sensors already provided within the Gem are implemented.
+If you intend to add your own sensor, it might be useful to look at how sensors already provided within the Gem are
+implemented.
 
 #### Robot Control
 
-The Gem comes with `ROS2RobotControlComponent`, which you can use to move your robot through [Twist](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Twist.msg) messages.
-The component subscribes to these messages on a configured topic. 
-You can use tools such as [rqt_robot_steering](https://index.ros.org/p/rqt_robot_steering/#galactic) to move your robot with Twist messages.
+The Gem comes with `ROS2RobotControlComponent`, which you can use to move your robot through:
+
+- [Twist](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Twist.msg) messages.
+- [AckermannDrive](https://index.ros.org/p/ackermann_msgs/#rolling)
+  The component subscribes to these command messages on a configured topic. The topic is "cmd_vel" by default, in a
+  namespace as dictated by ROS2Frame.
+
+To make use of received command messages, use either `AckermannControlComponent` or `RigidBodyTwistControlComponent`,
+depending on steering type. You can also implement your own control component or use LUA scripting to handle these
+commands.
+
+Unless scripting is used, control components should translate ROS 2 commands to events on `VehicleInputControlBus`.
+These events will be handled by a [`VehicleModelComponent`](#vehicle-model) if it is present.
+
+You can use tools such as [rqt_robot_steering](https://index.ros.org/p/rqt_robot_steering/#rolling) to move your robot
+with Twist messages.
+`RobotControl` is meant to be used with [ROS 2 navigation stack](https://navigation.ros.org/).
```

Comment:
Why is that? Is there anything in the gem especially tailored for the ROS2 nav stack? 

I believe it can be safely used with anything that publishes Twist or Ackermann commands :smiley:

Reply:
perhaps "It is suitable for use" - I believe not everyone knows that this is the case just based on topics